test-command.sh

```diff
@@ -3,12 +3,26 @@
 filename1=$1
 filename2=$2
 db_path=$3
-db_name=$4
-final_output=$5
-
+db_name_one=$4
+db_name_two=$5
+db_name_three=$6
+final_output=$7
 NPROCS=12
 
 filename1_short="test_res_R1"
 filename2_short="test_res_R2"
+filename_short="test_res"
 
-fastp --adapter_sequence GATCGGAAGAGCACACGTCTGAACTCCAGTCAC --adapter_sequence_r2 GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT -l 25 -i $filename1 -I $filename2 -w $NPROCS --stdout | minimap2 -ax sr -t $NPROCS $db_path/$db_name.mmi - -a | samtools fastq -@ $NPROCS -f 12 -F 256 -1 $final_output/${filename1_short}.trimmed.fastq.gz -2 $final_output/${filename2_short}.trimmed.fastq.gz
+# run trimming and two DB minimap2
+fastp --adapter_sequence GATCGGAAGAGCACACGTCTGAACTCCAGTCAC --adapter_sequence_r2 GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT -l 25 -i $filename1 -I $filename2 -w $NPROCS --stdout | minimap2 -ax sr -t $NPROCS $db_path/$db_name_one.mmi - -a | samtools fastq -@ $NPROCS -f 12 -F 256 | minimap2 -ax sr -t $NPROCS $db_path/$db_name_two.mmi - -a | samtools fastq -@ $NPROCS -f 12 -F 256 -1 $final_output/${filename1_short}.trimmed.fastq.gz -2 $final_output/${filename2_short}.trimmed.fastq.gz
+# run kraken2
+kraken2 --threads $NPROCS --db $db_path/$db_name_three --report $final_output/$filename_short-kraken2-report.txt --unclassified-out $final_output/$filename_short.kraken.trimmed.#.fastq --paired $final_output/${filename1_short}.trimmed.fastq.gz $final_output/${filename2_short}.trimmed.fastq.gz
```

Comment:
Maybe I'm missing something obvious, but I would have expected that somewhere we would just keep _not_ human reads but I don't see that command - is that necessary? I mean, kraken2 will profile everything in it's database (human or not) so I'm wondering if we are not deleting everything that kraken2 classified vs. only human.

Reply:
@antgonza the input database is only the human genomes. So anything not classified as human will be output as "unclassified". I also output the kraken2 report so that we have a record of how much kraken2 picks up after minimap2, which might be useful for QC/evaluation down the road.